CHANGELOG.md

```diff
@@ -29,6 +29,11 @@
   [Marcelo Fabri](https://github.com/marcelofabri)
   [#1271](https://github.com/realm/SwiftLint/issues/1271)
 
+* Exclude files defined in the configuration also when using the
```

Comment:
> Exclude files defined in the `excluded` value of SwiftLint's configuration when `--use-script-input-files` is specified.

Reply:
Also, this is a breaking change, so it should be in the "Breaking" section of the changelog.